packages/signature-controller/src/SignatureController.ts

```diff
@@ -164,6 +166,11 @@ export type SignatureControllerOptions = {
    * Callback to record the duration of code.
    */
   trace?: TraceCallback;
+
+  /**
+   * Api used to get decoding data for permits.
+   */
+  decodingApi?: string;
```

Comment:
Probably a future concern but we may want a callback such as `isDecodingEnabled` that allows the client to disable it per request.